### ISSUE TITLE
Request to keep "Pre-configured OpenCore Desktop EFI Folders" aligned with Dortania guide

### ISSUE THREAD

**Issue by chriswayg** [ic005k/OCAuxiliaryTools#139]
When setting up my system using `Desktop_10thGen_Comet_Lake_iMac20,2.plist` (iMac20,1.plist as well) I noticed some changes in the configuration which deviate from the Dortania guide for Comet Lake:

### ACPI - Add

https://dortania.github.io/OpenCore-Install-Guide/config.plist/comet-lake.html#acpi
![image](https://user-images.githubusercontent.com/13681327/152724118-3f0b35e3-10ef-47c2-82ce-3082c2033885.png)

In comparison, `Desktop_10thGen_Comet_Lake_iMac20,2.plist` added SSDT-PMC.aml which AFAIK is only needed for 300 series boards on Coffee Lake:

> SSDT-PMC - 300 series motherboards(non-Z370) don't declare the FW chip as MMIO in ACPI and so XNU ignores the MMIO region declared by the UEFI memory map. This SSDT brings back NVRAM support.
https://dortania.github.io/OpenCore-Install-Guide/config.plist/coffee-lake.html#acpi

![image](https://user-images.githubusercontent.com/13681327/152724561-4ad62e0b-600b-4362-8105-e041615e7595.png)

### ACPI - Quirks

Also `ResetLogoStatus` does not need to be checked IMHO.

### How to submit modification requests? 
We know, that the Dortania Guides will keep being developed and changed in the future and I believe it's important for the Base Config files to reflect the settings recommended there. Above are two example which I noticed. Should change requests for these configurations be directed to you or to the upstream author @5T33Z0

https://github.com/5T33Z0/OC-Little-Translated/tree/main/F_Desktop_EFIs

**Comment by 5T33Z0** [comment]
Thanks for your Feedback.

1. SSDT-RHUB: Not present in the latest base config on my repo. But I will re-add (disabled) an add a comment that it's only required for ASUS Boards.
2. SSDT-PMC: although a _necessity_ on 300 Series mainboards. On 400 Series or newer it's cosmetic only because the PMCR device is present in the IO Registry of iMac20,x. 

In general: the Dortania Guide provides the **basic** instructions to get the system running. My configs deviate in cases where I know from experience which settings can be tweaked/optimized. I have a 10th Gen Intel System and know what has to be present to run it better. There are errors in the dortania guide as well which I try to avoid such as that settings in the pictures do not match the actual required settings mentioned in the "More in-depth section".

In other words: I will deviate from the settings provided by the dortania guide where I think it's necessary or useful.

The latest versions of the base configs are on my repo https://github.com/5T33Z0/OC-Little-Translated/tree/main/F_Desktop_EFIs

**Comment by ic005k** [comment]
@5T33Z0 OK, added.

**Comment by chriswayg** [comment]
@5T33Z0 
> I will deviate from the settings provided by the dortania guide where I think it's necessary or useful.

You are right. There are certainly places where the dortania guide is not fully consistent or where new insights have not been incorporated yet. 

@5T33Z0, I have just looked through your Github repository and I truly appreciate all the valuable Tips and Tricks, Guides and material you have produced there. Thank you so much! 

Both you and @ic005k are doing amazing work which makes hackintoshing more fun and the systems a lot easier to maintain.

**Comment by ic005k** [comment]
@5T33Z0 I haven't released a new version yet, but if I do, this file will be there after you upgrade OCAT.
It currently exists in the database at
https://github.com/ic005k/OCAuxiliaryTools/blob/master/Database/EFI/OC/ACPI/SSDT-RHUB.aml
